views/question-table.pug

```diff
@@ -9,6 +10,12 @@ table.table
   tbody
     each q in questions
       tr
+        td
+          case q.show
```

Comment:
might be able to change this whole block to:
`input.checked(id='checked-' + q.id type='checkbox' checked=q.show)`

Reply:
Good point. I'll try it out and see how it goes.